contentful/web/theme/modules/SeoMetadata/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./SeaMetadata";
```

Comment:
**nitpick:** it should be `Seo` (typo in the filename)

**suggestion:** rename file + import